pkg/api/v1/types.go

```diff
@@ -437,8 +440,21 @@ const (
 
 	// MountOptionAnnotation defines mount option annotation used in PVs
 	MountOptionAnnotation = "volume.beta.kubernetes.io/mount-options"
+
+	// StorageTopologyConstraint defines topology constraints of a PV
+	// Value is type []TopologyConstraint
+	AlphaStorageTopologyConstraintsAnnotation = "volume.alpha.kubernetes.io/topology-constraints"
 )
 
+// TopologyConstraint contains label information that the resource is constrained to
+// A label key can have multiple values
+type TopologyConstraint struct {
+	// Key of the label
```

Comment:
OK. I'm just a bit weary of saying all local PVs need **some** topology constraints, without being prescriptive about it. It's an alpha API, so as you suggest we can bikeshed on that for now.